main.py

```diff
@@ -46,8 +46,8 @@
 class NoClientNetworkBlocksFound(Exception):
   def __init__(self, provider_name):
     Exception.__init__(self,
-                       'Could not find IP blocks associated with client provider {client_provider}.'.format(
-                           client_provider = provider_name))
+        ('Could not find IP blocks associated with client provider '+
+        '{client_provider}.').format(client_provider = provider_name))
```

Comment:
Don't need the `+` because Python will join the strings due to the parens. Need one more leading space on 50 to line up the quote characters:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Line_length#Line_length